lib/app/features/feed/views/components/user_info/user_info.dart

```diff
@@ -36,8 +37,18 @@ class UserInfo extends HookConsumerWidget {
         }
         return ListItem.user(
           onTap: () => ProfileRoute(pubkey: pubkey).push<void>(context),
-          title: Text(userMetadataEntity.data.displayName),
-          subtitle: Text(prefixUsername(username: userMetadataEntity.data.name, context: context)),
+          title: Text(
+            userMetadataEntity.data.displayName,
+            style: TextStyle(
```

Comment:
i think we should  apply style with `context.theme.appTextThemes`

Reply:
I pass optional `textColor` param to TextStyle, and it is used only to show white colors text on VideoPage black background. I think it is better to pass TextStyle? itself.